src/order/lattice.lean

```diff
@@ -32,6 +32,8 @@ of `sup` over `inf`, on the left or on the right.
   commutative, associative and satisfy a pair of "absorption laws".
 
 * `distrib_lattice`: a type class for distributive lattices.
+* `distrib_lattice.of_inf_sup_le`: Alternative constructor for `distrib_lattice` using the dual
+  distributive law.
```

Comment:
I think this would be more useful in the docstring for `distrib_lattice`; it's interesting for someone who is about to construct the instance, but not for someone looking at the file as whole.